### ISSUE TITLE
Jupyter notebook formatting stopped working

### ISSUE THREAD

**Issue by soumen-ghosh** [lensvol/intellij-blackconnect#80]
I have recently migrated to jupyterlab 3.6.*, jupyterlab_widgets 3.0.* and black 23.3.*. 

With these upgrades, on formatting trigger (like file save), all the cells of the notebook are getting merged and becoming a single markdown cells.

When I downgrade black to 22.8.0, jupyterlab to 3.2, formatting of notebook starts working again. I've not experimented if downgrading only black suffices or not.

I'm using PyCharm Professional 2022.2.2.

**Comment by lensvol** [comment]
Apparently it is the same issue as #72

**Comment by soumen-ghosh** [comment]
Yes, you can mark it as duplicate. Do you know of any timeline that this will be taken up?
